Roys/app/src/main/java/com/dwg_karrier/roys/Crawler.java

```diff
@@ -0,0 +1,120 @@
+package com.dwg_karrier.roys;
+
+import android.os.AsyncTask;
+import android.util.Log;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+/* Add on to listActivity
+* Use getTitle(), getContent(), getWordCount() */
+
+public class Crawler {
+  JSONObject jsonObject = null;
+
+  public Crawler(String strUrl) {
+    jsonObject = getJsonResponse(strUrl);
+  }
+
+  private JSONObject getJsonResponse(String pageUrl) {
+    try {
+      MyAsyncTask myAsyncTask = new MyAsyncTask();
+      jsonObject = myAsyncTask.execute(pageUrl).get();
+    } catch (Exception e) {
+      e.printStackTrace();
+    }
+    return jsonObject;
+  }
+
+  public String getTitle() {
+    if (this.jsonObject == null) {
+      return null;
+    }
+    String title;
```

Comment:
or if you want to concentrate the return logic into a single line,
`String title;`
`try {`
`  title = something`
`} catch (Exception e) {`
`  ...`
`  title = null;`
`}`
`return title;`
might be possible.